htdocs/mri/jiv/get_file.php

```diff
@@ -136,10 +136,10 @@
     $MimeType         = 'image/vnd.nrrd';
     $DownloadFilename = basename($File);
     break;
-case 'DICOMTAR':   
+case 'tar':   
```

Comment:
@christinerogers 

In the last commit, 
1) I switched it back to have the option to allow other tar to be downloaded (so I went back to dicomtar), but I changed the "trachive" to "DCM_" which is automatically appended by the pipeline (tarchive no longer exists in the archivelocation as it is now a relative location). 
2) ran phpcs